Makefile

```diff
@@ -95,6 +95,10 @@ build-local:
 	make _build-java
 	make _build-core
 	make _build-room
+	make _build-web
+
+_build-web:
+	yarn build:dst
```

Comment:
Before build should use yarn workspace focus install deps:
```bash
yarn workspaces focus @apitable/core @apitable/i18n-lang @apitable/icons @apitable/components @apitable/widget-sdk root
```

Reply:
why doesn't this exist in `make install`?